cw_monero/lib/monero_wallet_addresses.dart

```diff
@@ -27,6 +29,40 @@ abstract class MoneroWalletAddressesBase extends WalletAddresses with Store {
   @observable
   String address;
 
+  @override
+  String get latestAddress {
+    final accountIndex = 0;
+    var addressIndex = subaddress_list.numSubaddresses(accountIndex) - 1;
+    var address = getAddress(accountIndex: accountIndex,addressIndex: addressIndex);
+    while (hiddenAddresses.contains(address)) {
+      addressIndex++;
+      address = getAddress(accountIndex: 0, addressIndex: addressIndex);
+    }
+    return address;
+  }
+
+  @override
+  String get addressForExchange {
+    final accountIndex = 0;
+    var addressIndex = subaddress_list.numSubaddresses(accountIndex) - 1;
+    var address = getAddress(accountIndex: accountIndex,addressIndex: addressIndex);
+    while (hiddenAddresses.contains(address) || manualAddresses.contains(address)) {
```

Comment:
I believe both `latestAddress` and `addressForExchange` should check against both `hiddenAddresses` and `manualAddresses` so there shouldn't be a difference here and they can be merged into one getter `latestUnusedAddress` for example

Reply:
We shouldn't use manual addresses for exchange refunds, that's why there are 2 getters, we want to display manual addresses, but we don't want to use them for exchanges.